changes/2341-alecgerona.md

```diff
@@ -0,0 +1 @@
+fix(encoder): serialize NameEmail to str
```

Comment:
```suggestion
fix(encoder): serialize `NameEmail` to str
```
For the CI don't worry it's because of a the new sqlalchemy release. I opened https://github.com/encode/databases/pull/299 yesterday night to fix it

Reply:
Cool, thanks. Already updated.